src/main/python/main.py

```diff
@@ -187,23 +187,25 @@ def __init__(
         backup_experiment_start_index: int, 
         experiment_baseline_start_index: int, 
         experiment_baseline_end_index: int, 
-        test_plot_duration: float, 
+        test_pulse_plot_end: float, 
         test_pulse_baseline_samples: int,
         experiment_plot_bessel_order: int,
         experiment_plot_bessel_critical_frequency: float,
-        thumbnail_step: int
+        thumbnail_step: int,
+        test_pulse_plot_start: float
```

Comment:
Really minor nitpick but can you move this up to before test_pulse_plot_end parameter?